docs/source/array-numpy-compatibility.rst

```diff
@@ -0,0 +1,376 @@
+   :obj:`numpy.sinh`, :obj:`dask.array.sinh`, direct (ufunc)
+   :obj:`numpy.sort`, :obj:`dask.array.topk` [#4]_
+   :obj:`numpy.sort_complex`, \-
+   :obj:`numpy.source`, \-
+   :obj:`numpy.spacing`, :obj:`dask.array.spacing`, direct (ufunc)
+   :obj:`numpy.split`, \-
+   :obj:`numpy.sqrt`, :obj:`dask.array.sqrt`, direct (ufunc)
+   :obj:`numpy.square`, :obj:`dask.array.square`, direct (ufunc)
+   :obj:`numpy.squeeze`, :obj:`dask.array.squeeze`, dask equivalent
+   :obj:`numpy.stack`, :obj:`dask.array.stack`, dask equivalent
+   :obj:`numpy.std`, :obj:`dask.array.std` [#1]_, dask equivalent
+   :obj:`numpy.subtract`, :obj:`dask.array.subtract`, direct (ufunc)
+   :obj:`numpy.sum`, :obj:`dask.array.sum` [#1]_ [#2]_, dask equivalent
+   :obj:`numpy.swapaxes`, :obj:`dask.array.swapaxes`, dask equivalent
+   :obj:`numpy.take`, :obj:`dask.array.take` [#7]_, dask equivalent
+   :obj:`numpy.take_along_axis`, \-
+   :obj:`numpy.tan`, :obj:`dask.array.tan`, direct (ufunc)
+   :obj:`numpy.tanh`, :obj:`dask.array.tanh`, direct (ufunc)
+   :obj:`numpy.tensordot`, :obj:`dask.array.tensordot`, dask equivalent
+   :obj:`numpy.tile`, :obj:`dask.array.tile`, dask equivalent
+   :obj:`numpy.trace`, :obj:`dask.array.trace` [#5]_, dask equivalent
+   :obj:`numpy.transpose`, :obj:`dask.array.transpose`, dask equivalent
+   :obj:`numpy.trapz`, \-
+   :obj:`numpy.tri`, :obj:`dask.array.tri`, dask equivalent
+   :obj:`numpy.tril`, :obj:`dask.array.tril`, dask equivalent
+   :obj:`numpy.tril_indices`, :obj:`dask.array.tril_indices`, dask equivalent
+   :obj:`numpy.tril_indices_from`, :obj:`dask.array.tril_indices_from`, dask equivalent
+   :obj:`numpy.trim_zeros`, \-
+   :obj:`numpy.triu`, :obj:`dask.array.triu`, dask equivalent
+   :obj:`numpy.triu_indices`, :obj:`dask.array.triu_indices`, dask equivalent
+   :obj:`numpy.triu_indices_from`, :obj:`dask.array.triu_indices_from`, dask equivalent
+   :obj:`numpy.true_divide`, :obj:`dask.array.true_divide`, direct (ufunc)
+   :obj:`numpy.trunc`, :obj:`dask.array.trunc`, direct (ufunc)
+   :obj:`numpy.union1d`, :obj:`dask.array.union1d`, dask equivalent
+   :obj:`numpy.unique`, :obj:`dask.array.unique` [#16]_, dask equivalent
+   :obj:`numpy.unpackbits`, \-
+   :obj:`numpy.unravel_index`, :obj:`dask.array.unravel_index`, dask equivalent
+   :obj:`numpy.unwrap`, \-
+   :obj:`numpy.vander`, \-
+   :obj:`numpy.var`, :obj:`dask.array.var` [#1]_, dask equivalent
+   :obj:`numpy.vdot`, :obj:`dask.array.vdot`, dask equivalent
+   :obj:`numpy.vsplit`, \-
+   :obj:`numpy.vstack`, :obj:`dask.array.vstack` [#17]_, dask equivalent
+   :obj:`numpy.where`, :obj:`dask.array.where`, dask equivalent
+   :obj:`numpy.zeros`, :obj:`dask.array.zeros`, dask equivalent
+   :obj:`numpy.zeros_like`, :obj:`dask.array.zeros_like`, dask equivalent
+
+.. rubric:: Footnotes
+
+.. [#1] ``where`` parameter not supported.
+.. [#2] ``initial`` parameter not supported.
+.. [#3] Input must be a dask array.
```

Comment:
I am not sure about this note: this is how I have interpreted docstrings like for example  the [angle](https://docs.dask.org/en/stable/generated/dask.array.angle.html#dask.array.angle) one, where it is said that `array_like` are not supported.